addons/hr/hr.py

```diff
@@ -272,7 +272,7 @@ def _message_get_auto_subscribe_fields(self, cr, uid, updated_fields, auto_follo
 
 class hr_department(osv.osv):
     _name = "hr.department"
-    _description = "HR Department"
+    _description = "Department"
```

Comment:
@mart-e Yes you are right. 
but I thought if  in some-other module 'hr.department' is inherited and override _description attribute with different string then message changed.
-For hack-  there is already create method in hr.department.so I just suggest one more option 
to pass mail_create_nolog=True In context to use custom message and use message_post.